### PR TITLE
Update Logger in Core SDK

### LINKED ISSUE #16: Add logging
### What?

-

### Why?

-

### Engineering Section

-

### Blockers

_No response_

### PR DESCRIPTION
**Pull request by sameerdhulap**
## Issue
closes #16

## Describe your changes
Added os logger with SDK to log info in client App

## How to test
Test Sample app for various log  level expose

## Checklist:

- [x] My code follows the style guidelines for this repo
- [x] I have performed a self-review of my code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings/errors
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I don't require ops changes for this PR to go to prod
- [x] This change does not include a migration

### CODE REVIEW

File: Sources/WoosmapGeofencing/Business Logic/Distance.swift [hunk 1]
```diff
@@ -142,7 +143,13 @@ public class Distances {
                     }
                 }
             } else {
-                print("WoosmapGeofencing.DistanceAPIData " + jsonStructure.status!)
+                if(WoosLog.isValidLevel(level: .info)){
```
Comment (strider1981) [inline Sources/WoosmapGeofencing/Business Logic/Distance.swift]: These if conditions are repeated many times? Can't they be put in a single function?

**Review by strider1981 — CHANGES_REQUESTED** [review]

File: Sources/WoosmapGeofencing/Business Logic/Distance.swift (hunk 1 above)
Comment (sameerdhulap) [reply Sources/WoosmapGeofencing/Business Logic/Distance.swift]: it is not in a loop. i have modified log level to warning

**Review by strider1981 — APPROVED** [review]
LGTM

**Comment by sameerdhulap** [comment]
4.1.0 release

**Pull request merged**